src/Products/Signature/Loader/SignatureLoader.cs

```diff
@@ -264,9 +259,9 @@ public List<SignatureFileDescriptionEntity> LoadTextSignatures(string xmlFolder)
 
                 return fileList;
             }
-            catch (Exception ex)
+            catch (Exception)
```

Comment:
Codacy found an issue: [Add logic to this catch clause or eliminate it and rethrow the exception automatically.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Signature-for-.NET-WebForms/pullRequest?prid=6336225)